qutebrowser/browser/qtnetworkdownloads.py

```diff
@@ -198,21 +199,23 @@ def _after_set_filename(self):
 
     def _ask_confirm_question(self, title, msg):
         no_action = functools.partial(self.cancel, remove_data=False)
+        url = 'file://{}'.format(self._filename)
```

Comment:
I'm not sure whether we should really yank a `file://` URL here. Yanking the filename instead would be a bit unorthodox, but IMHO much more useful to the user. What do you think?

Reply:
I think you may be right about the usefulness, but then we would need to rename question.url to question.data since it wouldn't be a URL anymore.

A filename can only be a URL with a "file://" scheme, no?

Reply:
Yeah, I guess we'd need to rename it to `data` or `yank_data` or something again then. Then again, that makes everything harder to explain (in docstrings etc.). Let's keep it as-is and see what people say.